repose-aggregator/src/docs/asciidoc/services/index.adoc

```diff
@@ -3,10 +3,10 @@
 include::../_includes/in-progress.adoc[]
 
 == <<atom-feed-consumption.adoc#,Atom Feed Consumption Service>>
-*IN PROGRESS*
+Enables a *Repose* filter to read an Atom feed.
```

Comment:
Technically, another service could also use this service to consume an Atom feed.

Maybe something like:
> Consolidates the consumption of Atom feeds, and provides a simple interface to retrieve consumed data.

Reply:
Updated.